packages/nimble-components/src/units/volt/index.ts

```diff
@@ -0,0 +1,28 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { template } from '../base/template';
+import { UnitScale } from '../base/unit-scale';
+import type { UnitScaleFormatterConstructor } from '../../table-column/number-text/models/unit-scale-formatter';
+import { VoltScaleFormatter } from '../../table-column/number-text/models/volt-scale-formatter';
```

Comment:
number-text isn't responsible for all formatters, just shared base classes. formatters provided by the unit elements should be organized with them
`/units/volt/models/volt-scale-formatter`